pyhoma/client.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, username: str, password: str, api_url: str = API_URL) -> None
         self.__roles = None
         self.event_listener_id: Optional[str] = None
 
-        self.session = aiohttp.ClientSession()
+        self.session = aiohttp.ClientSession(headers={"Connection": "keep-alive"})
```

Comment:
Not related, but would it make sense to also add a User Agent and make this configurable on instantiation?